### PR TITLE
Release V0.2.5

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Release the latest bug fixes and updates
- Environment variable management to vault plugin
- Localist widget theming
- adplug and localist embed CSP allowance (netlify toml)
- SAA Story Card revisions
- Travel Study SEO document title fix
- Changes to redirects

# Review By (Date)
- Friday February 18th

# Criticality
- It contains some important updates to the client so EOW is semi urgent

# Review Tasks

## Setup tasks and/or behavior to test

1. Review code
2. Spot check preview site for irregularities 
3. Ensure story card still works as it was intended
4. Ensure that adplug/localist work (pages TBD)
5. Ensure that travel study gets the correct document title

@yvonnetangsu Any changes to Storyblok required for localist or the story card?

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@
   for = "/*"
   [headers.values]
     Access-Control-Allow-Origin = "*"
-    Content-Security-Policy = "default-src https: 'unsafe-inline'; form-action https:; img-src https: data:"
+    Content-Security-Policy = "default-src https: 'unsafe-inline' 'unsafe-eval'; form-action https:; img-src https: data:"
```
Comment (mattanglin) [inline netlify.toml:18]: I assume this is because of 3rd party lib dependencies (the adplug/localist?)? 😐

**Review by mattanglin — APPROVED** [review]
lgtm 👍🏼

File: netlify.toml (hunk 1 above)
Comment (sherakama) [reply netlify.toml:18]: Yeah, we allow embedding from just about anywhere but do restrict the embed component to certain roles.

Comment (sherakama) [reply netlify.toml:18]: This gives me the shivers as well but unless we curate a list of acceptable 3rd parties we are stuck leaving it wide open.

**Comment by yvonnetangsu** [comment]
@sherakama Yup, we're going to need to add the manual story card to the PROD Storyblok space. I'd be happy to do that.

**Comment by yvonnetangsu** [comment]
saaStoryCard (manual story card) component added on PROD. 
Note: I had to do this manually as there are a bunch of still in development components on the DEV SAA Storyblok space so I didn't want to use the CLI tool to push all the components from DEV to PROD 😬

**Pull request merged**